src/inspect_ai/agent/_human/state.py

```diff
@@ -1,4 +1,4 @@
-from time import time as current_time
+import time as python_time
```

Comment:
I struggled to mock `time.time` when it was imported from `time` as `current_time`. Importing the entire module made it easy to mock.